implementations/rust/ockam/ockam_identity/src/secure_channel/encryptor_worker.rs

```diff
@@ -312,17 +284,35 @@ impl EncryptorWorker {
 
     async fn send_close_channel(&mut self, ctx: &Context) -> Result<()> {
         let msg = SecureChannelMessage::Close;
+        let msg = Self::add_padding(msg);
 
         // Encrypt the message
         let msg = self.encrypt(ctx, msg).await?;
 
         let remote_route = self.shared_state.remote_route.read().unwrap().route.clone();
         // Send the message to the decryptor on the other side
-        ctx.send_from_address(remote_route, msg, self.addresses.encryptor.clone())
-            .await?;
+        ctx.send_from_address(
+            remote_route,
+            NeutralMessage::from(msg),
+            self.addresses.encryptor.clone(),
+        )
+        .await?;
 
         Ok(())
     }
+
+    fn add_padding(msg: SecureChannelMessage) -> SecureChannelPaddedMessage {
+        // Naїve padding of 0 to 255 zeros
+        // let padding_length: u8 = ockam_core::compat::rand::random();
```

Comment:
Why is the padding commented out for now? Would that be possible to add a test to show that messages are effectively padded, to make sure that we don't have regressions there?

Reply:
We should pick the strategy for padding, which I think we'll do later. For now I only added a field so that we can add padding without any compatibility issues later. Adding a test is problematic since the API for padding is not exposed

Reply:
Can't we test something that when we encrypt the same message twice, the on-wire bytes are likely to have a different size?

Reply:
Why would it be different if we don't add any padding at the moment?

Reply:
I meant when padding is used of course :-). I was responding to your comment:
> Adding a test is problematic since the API for padding is not exposed

Reply:
Yes, so currently padding is always empty and you can't use it. So it's hard to test